intermediate/challenge-1/src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,23 @@
+spring:
+  config:
+    activate:
+      on-profile: dev
+  dev:
+    console:
+      enabled: true
+      path: /h2-console
+  datasource:
+    url: jdbc:h2:mem:bus-app;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+    username: sa
+    password:
+    driverClassName: org.h2.Driver
+  jpa:
+    database-platform: org.hibernate.dialect.H2Dialect
+    hibernate:
+      ddl-auto: update
```

Comment:
how your set-up would change if you have to use another database or different kind ?

Reply:
for example , "org.hibernate.dialect.PostgresSQLDialect" is for Postgressql.
based on environment we can create different yml files and set that as spring profile.
ddl-auto should be none if we are connecting with an existing db.
The current configuration is for h2 db and it will create tables based on entities.